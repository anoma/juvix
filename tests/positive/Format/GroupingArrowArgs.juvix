module GroupingArrowArgs;
  open import Stdlib.Prelude;

  -- grouping of type arguments
  type AnExampleType (A : Type) :=
    | anExampleType : {A : Type} -> AnExampleType;

  exampleFunction : {A : Type}
      -> (List A
        -> (List A
          -> (List A
            -> (List A -> (List A -> (List A -> (List A -> Nat)))))));
  exampleFunction _ _ _ _ _ _ _ := 1;
end;
