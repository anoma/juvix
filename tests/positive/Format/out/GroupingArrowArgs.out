module GroupingArrowArgs;
  open import Stdlib.Prelude;

  -- grouping of type arguments
  type AnExampleType (A : Type) :=
    | anExampleType : {A : Type} -> AnExampleType;

  funOnAnExampleType : {A : Type} -> List A -> List A -> List A -> List A -> List A -> List A -> List A -> Nat;
  funOnAnExampleType := const (const (const 0));
end;
